packages/pipeline-editor/src/RuntimesWidget.tsx

```diff
@@ -75,23 +75,30 @@ class RuntimesDisplay extends MetadataDisplay<
       }
     }
 
-    if (metadata.schema_name === 'airflow' && metadata_props) {
-      const githubRepoUrl =
-        getGithubURLFromAPI(metadata.metadata.github_api_endpoint) +
-        metadata.metadata.github_repo +
-        '/tree/' +
-        metadata.metadata.github_branch +
-        '/';
-      githubRepoElement = (
-        <span>
-          <h6>{metadata_props.github_repo.title}</h6>
-          <a href={githubRepoUrl} target="_blank" rel="noreferrer noopener">
-            {githubRepoUrl}
-          </a>
-          <br />
-          <br />
-        </span>
-      );
+    if (metadata_props) {
+      if (metadata.schema_name === 'airflow') {
+        const githubRepoUrl =
+          getGithubURLFromAPI(metadata.metadata.github_api_endpoint) +
+          metadata.metadata.github_repo +
+          '/tree/' +
+          metadata.metadata.github_branch +
+          '/';
+        githubRepoElement = (
+          <span>
+            <h6>{metadata_props.github_repo.title}</h6>
+            <a href={githubRepoUrl} target="_blank" rel="noreferrer noopener">
+              {githubRepoUrl}
+            </a>
+            <br />
+            <br />
+          </span>
+        );
+      } else if (metadata.schema_name === 'kfp') {
```

Comment:
since this new if block doesn't use `metadata_props` there's no reason to tie into this if block. It can just be a separate if block